test/Microsoft.IdentityModel.TestUtils/IdentityComparer.cs

```diff
@@ -33,54 +33,55 @@ namespace Microsoft.IdentityModel.TestUtils
 {
     public class IdentityComparer
     {
+        // Dictionary of types and the validation function.
```

Comment:
>Dictio [](http://example.com/codeflow?start=11&length=6)

wondering if this issue: [https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/2148](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/2148) will be fixed now?